src/pages/Facility/settings/billing/layout.tsx

```
@@ -90,6 +91,18 @@ export function BillingSettingsLayout() {
     basePath: `/facility/${facilityId}/settings/billing`,
     routeProps: { facilityId },
   });
+  const fullPath = useFullPath();
+  const fullPathMap = useMemo(
+    () =>
+      fullPath.split("/").reduce(
+        (acc, part) => ({
+          ...acc,
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)